WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -49,6 +49,15 @@
         expect(response).to render_template :new
       end
     end
+
+    context 'when signed in as a regular user' do
+      sign_in { FactoryGirl.create :user }
+
+      it 'doesn\'t allow access' do
+        get :new
```

Comment:
Even more important than testing `:new` is to make sure that users aren't allowed to post to `:create`. Could we add a test for that?